charts/project-origin-wallet/values.yaml

```diff
@@ -119,3 +119,7 @@ persistance:
 
     # storage defines the storage configuration for the database
     size: 10Gi
+
+  # Add BYOD configuration
+  byod:
+    secretName: ""  # The user must create a secret with the necessary DB connection information
```

Comment:
@sahma19 good practice would be to also enable the secretKey be defined by the user.